backend/shop/serializers.py

```diff
@@ -62,13 +62,23 @@ class Meta:
         model = OrderItem
         fields = ["id", "order", "product", "quantity"]
 
```

Comment:
read_only = ["order"]

Ten tink about if you really need order = PrimaryRelatedField(). Try wit read_only and ten see wat you get. If it does not work, look for te least-code solution.


mapping fields: https://www.django-rest-framework.org/api-guide/serializers/#customizing-field-mappings